frontend/pages/hosts/details/cards/Software/Software.tsx

```diff
@@ -193,19 +190,6 @@ const SoftwareTable = ({
     router.push(path);
   };
 
-  const renderVulnFilterDropdown = () => {
```

Comment:
Is there no more vulnerable filtering on the host details page?

Reply:
it was removed in the figma but I will confirm to make sure. I will add it back if it was not meant to be removed